projects/observability/src/shared/components/top-n/top-n-chart.component.ts

```diff
@@ -63,20 +63,20 @@ export class TopNChartComponent implements OnChanges {
 
     this.itemOptions = this.data
       .sort((first, second) => second.value - first.value)
-      .map(
-        (option): TopNItemOptions => ({
-          label: option.label,
-          width: `${((option.value / maxValue!) * 100).toFixed(2)}%`,
-          value: option.value
-        })
-      );
+      .map(option => ({
+        label: option.label,
+        width: `${((option.value / maxValue!) * 100).toFixed(2)}%`,
+        value: option.value,
+        original: option
+      }));
   }
 }
 export interface TopNData {
   label: string;
   value: number;
 }
 
-interface TopNItemOptions extends TopNData {
+interface TopNItemOption<T extends TopNData> extends TopNData {
   width: string;
+  original: T;
```

Comment:
What's the reason for this change?

Reply:
The previous API had no way of disambiguating two objects with identical labels.

Reply:
I see. Thanks!